AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -164,13 +164,16 @@ def AtomEditorComponents_Light_AddedToEntity():
     from Atom.atom_utils.atom_constants import (AtomComponentProperties, LIGHT_TYPES, INTENSITY_MODE,
                                                 ATTENUATION_RADIUS_MODE, SHADOWMAP_SIZE, SHADOW_FILTER_METHOD)
 
-
     with Tracer() as error_tracer:
         # Test setup begins.
         # Setup: Wait for Editor idle loop before executing Python hydra scripts then open "Base" level.
         TestHelper.init_idle()
         TestHelper.open_level("Graphics", "base_empty")
 
+        # Copy LIGHT_TYPES to pop the 'unknown' value and prevent it from being run in this test.
```

Comment:
pop the 'unknown' key
value is the numeric value unknown represent or zero. pop is removing the entire key-value pair

this dict copy should also be closer to the for loop that uses it so its clearly associated

Reply:
Changed to 'unknown' value and moved closer to the loop.